utils/rsaencryption/rsa_encryption.go

```diff
@@ -121,3 +135,22 @@ func ExtractPublicKey(sk *rsa.PrivateKey) (string, error) {
 
 	return base64.StdEncoding.EncodeToString(pemByte), nil
 }
+
+// ExtractKeysFromFile extract rsa private key from pem file with password encryption
+func ExtractKeysFromFile(password string) (string, error) {
+	pemData, err := ioutil.ReadFile("private_key.pem")
```

Comment:
the filename should be read from the config.yaml. also I think it's better that the whole file reading logic will happen in `cli/operator/node.go` and not here. than you just pass encrypted read data and password to this func, or just use `ConvertPemToPrivateKey` and this won't be needed.